blueman/plugins/manager/Services.py

```diff
@@ -53,12 +53,8 @@ def add_menu_item(manager_menu, service):
             item.show()
 
         for service in get_services(device):
-            try:
+            if service.group == 'network':
```

Comment:
I think we should drop the Input and Audio service classes as they are not actually used now and make `connect`, `disconnect`, `connected` abstract in `blueman.Service`. No need to check for specific service types here then as `get_services` will only return relevant ones.